database-commons/src/main/java/co/cask/db/batch/sink/DBSink.java

```diff
@@ -130,15 +132,7 @@ private void setColumnsInfo(List<Schema.Field> fields) {
       .map(Schema.Field::getName)
       .collect(collectingAndThen(toList(), Collections::unmodifiableList));
 
-    dbColumns = null;
-    if (columns.size() > 1) {
-
-      dbColumns = String.join(",", columns);
-
-    } else if (columns.size() == 1) {
-      dbColumns = columns.get(0);
-    }
-
+    dbColumns = String.join(",", columns);
```

Comment:
this is not what I commented about. You can using the joining collector above in the stream.

Reply:
I'm not sure I understand what do you mean. We have two fields - List columns and String dbColumns. columns list is used several times for validating schema while dbColumns string is used for OutputFormat configuration. 

Sure, I can use joining collector to assign dbColumns but then I'll need to split the string in order to validate schema.

Reply:
oh didn't realize these were different variables. Ignore this comment